reactor-netty-core/src/test/java/reactor/netty/tcp/TcpServerTests.java

```diff
@@ -1132,8 +1128,8 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) {
 		    .receive()
 		    .blockLast(Duration.ofSeconds(30));
 
-		assertNotNull(hostname.get());
-		assertEquals("test.com", hostname.get());
+		assertThat(hostname.get()).isNotNull();
+		assertThat(hostname.get()).isEqualTo("test.com");
```

Comment:
the assertions can be fluently chained (technically the `isNotNull` could even be omitted as it is implicitely checked by `isEqualTo`)

```suggestion
		assertThat(hostname.get())
		    .isNotNull()
		    .isEqualTo("test.com");
```